### DISCUSSION TITLE
license and copyright

### DISCUSSION THREAD

**Discussion by Veerhan-glitch** [community/community#39588] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi the thing i would like to know is that can i add the following thing to the README file of my repo or not,
[A bit of background, the repo (my own not a fork) has a MIT LICENCE (unedited except the name and year thing)]

"This repo contains a MIT LICENCE which states that you are allowed to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions: 
 The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
 
 NOTE: For better understanding of the above read the LICENSE provided. The above is not the license or modified version of the provided LICENSE, it is only a gist of what you are allowed to do.

The only additional thing to be followed is to give a proper credit/mention of the place where you got the copy from and/or the original project. "

Also just to make sure, if someone forks my repo, they have to include an un edited part of my copyright statements (the above part) and the provided license right?

**Comment by airtower-luna** [comment, marked as answer]
There's nothing that prevents you from adding a statement like that, but I wouldn't recommend it. The reason is that it's basically paraphrasing the license, which is more likely to cause confusion than clarity, because then you have *two* different statements on what is and isn't allowed. I'd stick to mentioning the license, maybe a one sentence summary if you want ("in short, that means you can use the code as you like, but you have to keep my copyright and license notice").

The copyright and license statement is usually the actual license file, and license headers in your source files, the part where it states your copyright and the actual license. I assume copyright lawyers could fill many papers with what is or isn't a valid license statement, but that's how I as a non-lawyer and software developer would understand it. :wink:

**Reply by Veerhan-glitch** [reply]
Hmm got it thx for for answering
